src/librustc_mir/interpret/cast.rs

```diff
@@ -118,17 +118,15 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
             }
             // The rest is integer/pointer-"like", including fn ptr casts and casts from enums that
             // are represented as integers.
-            _ => {
-                assert!(
-                    src.layout.ty.is_bool()
-                        || src.layout.ty.is_char()
-                        || src.layout.ty.is_enum()
-                        || src.layout.ty.is_integral()
-                        || src.layout.ty.is_any_ptr(),
-                    "Unexpected cast from type {:?}",
-                    src.layout.ty
-                )
-            }
+            _ => assert!(
```

Comment:
It is really disappointing that I cannot stop rustfmt from doing silly things like this. :( Cc https://github.com/rust-lang/rustfmt/issues/3995